lib/models/Posts.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true
+  },
+  photoUrl: {
+    type: String,
+    required: true
+  },
+  caption: {
+    type: String,
+    required: true
+  },
+  tags: [String]
```

Comment:
Other way you could have hit line 17 that would allow room for things like 'require':

tags: [{
   type: String,
   required: true
}]